app/models/post.rb

```diff
@@ -1,3 +1,17 @@
 class Post < ApplicationRecord
-  belongs_to :author, class_name: 'User'
+  after_save :post_increment
+
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :likes
+  has_many :comments
+
+  def recent_comment
+    comments.where(author_id.params[:id]).order(created_at: :desc).limit(5)
+  end
+
+  private
+
+  def post_increment
+    author.increment!(:posts_counter)
+  end
```

Comment:
✅ Post model has a method that updates the posts counter for a user 👍🏻

Reply:
Thanks for your review, suggested changes implemented.